website/docs/r/site_recovery_vmware_replicated_vm.html.markdown

```diff
@@ -0,0 +1,216 @@
+  default_recovery_disk_type                 = "Standard_LRS"
+  target_network_id                          = azurerm_virtual_network.example.id
+
+  network_interface {
+    source_mac_address = "00:00:00:00:00:00"
+    target_subnet_name = azurerm_subnet.example.name
+    is_primary         = true
+  }
+
+  depends_on = [azurerm_site_recovery_vmware_replication_policy_association.example]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the replicated VM. Changing this forces a new resource to be created.
+
+* `recovery_vault_id` - (Required) The ID of the Recovery Services Vault where the replicated VM is created. Changing this forces a new resource to be created.
+
+* `source_vm_name` - (Required) The name of the source VM in VMWare. Changing this forces a new resource to be created.
+
+* `appliance_name` - (Required) The name of VMWare appliance which handles the replication. Changing this forces a new resource to be created.
+
+* `recovery_replication_policy_id` - (Required) The ID of the policy to use for this replicated VM. Changing this forces a new resource to be created.
+
+* `physical_server_credential_name` - (Required) The name of credential to access source VM. Changing this forces a new resource to be created. More information about the credentials could be found [here](https://learn.microsoft.com/en-us/azure/site-recovery/deploy-vmware-azure-replication-appliance-modernized).
+
+* `license_type` - (Optional) The license type of the VM. Possible values are `NoLicenseType`, `NotSpecified` and `WindowsServer`. Defaults to `NotSpecified`.
+
+* `target_resource_group_id` - (Required) The ID of resource group where the VM should be created when a failover is done. Changing this forces a new resource to be created.
+
+* `target_vm_name` - (Required) Name of the VM that should be created when a failover is done. Changing this forces a new resource to be created.
+
+* `default_log_storage_account_id` - (Optional) The ID of the stroage account that should be used for logging during replication. 
+
+**Note:** Only standard types of stroage accounts are allowed.
+
+**Note:** At least one of `default_log_storage_account_id` or `managed_disk` must be specified.
+
+**Note:** Changing `default_log_storage_account_id` forces a new resource to be created. But removing it does not.
+
+* `default_recovery_disk_type` - (Optional) The type of storage account that should be used for recovery disks when a failover is done. Possible values are `Standard_LRS`, `Standard_LRS` and `StandardSSD_LRS`. 
+
+**Note:** At least one of `default_recovery_disk_type` or `managed_disk` must be specified.
+
+**Note:** Changing `default_recovery_disk_type` forces a new resource to be created. But removing it does not.
+
+* `target_network_id` - (Optional) The ID of network to use when a failover is done.
+
+* `target_subnet_name` - (Optional) Name of the subnet to use when a failover is done.
```

Comment:
The `target_subnet_name` has been removed, it could be specified in `network_interface` now. In this case, I don't think they could be combined toghter..